velox/dwio/parquet/writer/arrow/CMakeLists.txt

```diff
@@ -45,7 +45,6 @@ velox_link_libraries(
   velox_dwio_arrow_parquet_writer_lib
   velox_dwio_arrow_parquet_writer_util_lib
   velox_dwio_parquet_common
-  velox_dwio_parquet_thrift
```

Comment:
We don't need this because `velox_dwio_parquet_common` already depends on `velox_dwio_parquet_thrift`.